src/simplewallet/simplewallet.cpp

```diff
@@ -1035,18 +1158,22 @@ bool simple_wallet::print_fee_info(const std::vector<std::string> &args/* = std:
   {
     if (priority == fee_priority::Default)
       continue;
-    uint64_t mult = m_wallet->get_fee_multiplier(priority);
+    uint64_t mult = m_wallet_impl->getFeeMultiplier(tools::fee_priority_utilities::as_integral(priority), /* fee_algorithm */ -1);
```

Comment:
Would it be easy to use the enum constant `Unset` instead of -1?

(Did this code come in through a merge / rebase and so isn't yours?)